Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -203,8 +202,7 @@ private void CreateSoundDaytime(PxVobSoundDaytimeData vob)
             if (!FeatureFlags.I.EnableSounds)
                 return;
             
-            var vobObj = soundCreator.Create(vob, parentGos[vob.type]);
-            SetPosAndRot(vobObj, vob.position, vob.rotation!.Value);
```

Comment:
We need the position to be set before adding the item to the grid, so that we can actually see in which grid cell the gameobject is
if we do it after the gameobject is created, the gameobject will be set at cell 0,0,0